x-pack/solutions/security/plugins/security_solution/public/asset_inventory/hooks/use_dynamic_entity_flyout.test.ts

```diff
@@ -11,7 +11,6 @@ import { useExpandableFlyoutApi } from '@kbn/expandable-flyout';
 import { useKibana } from '../../common/lib/kibana';
 import { useOnExpandableFlyoutClose } from '../../flyout/shared/hooks/use_on_expandable_flyout_close';
 import {
-  UniversalEntityPanelKey,
```

Comment:
please remove front end changes from this PR, while removing universal entity is one thing, universal flyout panel is still a thing and should not be removed from code or tests.
universal flyout will be presented for all entities which do not have a matching flyout of their own. for now they are just user, host and service. the rest will use the universal flyout.

change for the entity.type from universal to something else like s3 is already been done on my side as well